src/dfx-core/src/config/model/dfinity.rs

```diff
@@ -429,6 +430,18 @@ pub struct InitializationValues {
     /// A setting of 0 means that the canister will trap if it tries to allocate new storage while the subnet's memory usage exceeds 450 GiB.
     #[schemars(with = "Option<u128>")]
     pub reserved_cycles_limit: Option<u128>,
+
+    /// # WASM Memory Limit
+    /// Specifies a soft limit (in bytes) on the Wasm memory usage of the canister.
+    ///
+    /// Update calls, timers, heartbeats, installs, and post-upgrades fail if the
+    /// WASM memory usage exceeds this limit. The main purpose of this setting is
+    /// to protect against the case when the canister reaches the hard 4GiB
+    /// limit.
+    ///
+    /// Must be a number between 0 and 2^48 (i.e. 256 TiB), inclusive.
+    #[schemars(with = "Option<u64>")]
+    pub wasm_memory_limit: Option<Byte>,
```

Comment:
The type and schemar of this filed seems to be consistent with the existing `memory_allocation` field above.

But it is not consistent with the CLI flag `--wasm-memory-limit` below.

Can we have a unified description on the input format for this config field and the CLI flag?